WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md

```diff
@@ -0,0 +1,401 @@
+
+    Type **GPMC.msc**, and then press ENTER.
+
+    > [!WARNING]
+    > <BR>You can link a new GPO to the domain or to any organizational unit (OU). The GPO modifications implemented in this document alter the registry settings of the affected computers. You can't undo these settings by deleting or unlinking the GPO. The settings can only be undone by reversing them in the GPO settings or by modifying the registry using another technique.
+
+1. In the Group Policy Management console, expand the **Forest** object, expand the **Domains** object, and then expand the specific domain that contains the computer accounts that you want to change. If you have a specific OU that you want to modify, then navigate to that location. select an existing GPO or right-select and then select **Create a GPO in this domain, and Link it here** to create a new GPO. Right-select the GPO you want to modify and then select **Edit**.
+
+1. In the navigation pane, under **Computer Configuration**, expand **Policies**.
+
+1. Right-select **Administrative Templates**, and then select **Add/Remove Templates**.
+
+1. In **Add/Remove Templates**, select **Add**. In the **Policy Templates** dialog box, select the .adm template that you previously saved. select **Open**, and then select **Close**.
+
+1. In the navigation pane, expand **Administrative Templates**, and then expand **Classic Administrative Templates (ADM)**.
+
+1. elect **Windows AutoUpdate Settings**, and in the details pane, double-select **URL address to be used instead of default ctldl.windowsupdate.com**.
+
+1. Select **Enabled**. In the **Options** section, enter the URL to the file server or web server that contains the CTL files. For example, `https://server1/CTL` or `file://\\\\server1\\CTL`. select **OK**. Close the Group Policy Management Editor.
+
+The policy is effective immediately, but the client computers must be restarted to receive the new settings, or you can type **gpupdate /force** from an elevated command prompt or from Windows PowerShell.
+
+> [!IMPORTANT]
+> The trusted and untrusted CTLs can be updated on a daily basis, so ensure that you keep the files synchronized by using a scheduled task or another method (such as a script that handles error conditions) to update the shared folder or web virtual directory. For additional details about creating a scheduled task, see [Schedule a Task](https://technet.microsoft.com/library/cc748993.aspx). If you plan to write a script to make daily updates, see the [New Certutil Options]() and [Potential errors with Certutil -SyncWithWU]() sections of this document. These sections provide more information about command options and the error conditions.
+
+## Redirect the Microsoft Automatic Update URL for untrusted CTLs only
+
+Some organizations may want only the untrusted CTLs (not the trusted CTLs) to be automatically updated. To accomplish this, you can create two .adm templates to add to Group Policy.
```

Comment:
`To accomplish this` - This isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?